actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Make sure local dev/prod puma configs do not clobber `ActionDispatch::SystemTesting`-preferred defaults
+
+    *Max Schwenk*
```

Comment:
Can you clarify what settings changed? Something like "Ensure local dev and prod puma configs do not clobber  `ActionDispatch::SystemTesting` defaults. Adds `workers: 0` and `daemon: false`.